runtime/doc/options.txt

```diff
@@ -3267,7 +3267,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 				     >:SignColumn,B:SpellBad,P:SpellCap,
 				     R:SpellRare,L:SpellLocal,-:Conceal,
 				     +:Pmenu,=:PmenuSel,x:PmenuSbar,
-				     X:PmenuThumb")
+				     X:PmenuThumb,q:QuickFixLine")
```

Comment:
What is that option used for anyway? I guess it's supposed to be used for temporarily "redirecting" highlight groups. Opposed to `:highlight link From To`, where you have to remember the old state of `From` if you want to go back later.

That said, neither the runtime files nor anyone of the many Vimmer's configs I hoard uses that option.